lib/client-assets.php

```diff
@@ -966,7 +895,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$script .= <<<JS
 		window._wpLoadGutenbergEditor = wp.api.init().then( function() {
 			wp.blocks.registerCoreBlocks();
-			return wp.editPost.initializeEditor( 'editor', window._wpGutenbergPost, editorSettings );
+			return wp.editPost.initializeEditor( 'editor', window._wpGutenbergPostId, editorSettings, window._wpGutenbergDefaultPost );
```

Comment:
Can you simply inline `_wpGutenbergPostId` and `_wpGutenbergDefaultPost`?

Reply:
Not sure what you mean? The reason those are different and not at the same position is that one is mandatory while the other is optional and could probably be removed later if we remove the demo content.

Reply:
Why not wrap this code with `sprintf` and inject those 2 variables directly in here instead of having globals?

Reply:
It's difficult to express with PHP but simplified version would be:
```php
$init_script = <<<JS
    ...
    return wp.editPost.initializeEditor( 'editor', %s, editorSettings, %s );
    ...
JS;
$script .= sprintf( init_script, $post->ID, wp_json_encode( array( 'title' => ... ) ) );
````

Reply:
Gotcha, `window._wpGutenbergDefaultPost` can be undefined in some cases but I guess it's feasible.

Reply:
Ok, this doesn't work easily because of the content of the demo post is a JavaScript file. We could try to convert it to JSON but seems not worth it.